src/utils/utils_cleaner.c

```diff
@@ -340,8 +342,10 @@ static int _ocf_cleaner_update_metadata(struct ocf_request *req)
 				&core_id, &req->part_id);
 		req->core = &cache->core[core_id];
 
+		ocf_collision_start_exclusive_access(&cache->metadata.lock, page);
```

Comment:
It seems that exclusive access is not required here.

Reply:
fixed